website/templates/emails/comment_replies.txt.mako

```diff
@@ -1,3 +1,10 @@
 ${user} replied to your comment "${parent_comment}" on your project "${title}": ${content}.
 
-${'\t'}To view this on the Open Science Framework, please visit: ${url}.
\ No newline at end of file
+${'\t'}To view this on the Open Science Framework, please visit: ${url}.
+
+
+Center for Open Science
+210 Ridge McIntire Road
+Suite 500
+Charlottesville, VA 22903-5083
+Privacy Policy: https://github.com/CenterForOpenScience/cos.io/blob/master/PRIVACY_POLICY.md
```

Comment:
Ugh. Yeah, we need to have actual templates for this. I believe there's a ticket to take care of this at some point.